src/store/reducers/healthcheckInfo.ts

```diff
@@ -0,0 +1,122 @@
+import _ from 'lodash';
```

Comment:
I'd get rid of lodash where it is possible
But more importantly I'd get rid of these kinds of lodash imports. The package supports tree-shaking by using explicit imports like the ones below. This one imports the whole lib.